### DISCUSSION TITLE
Ideas for Tournaments/Events on the server

### DISCUSSION THREAD

**Discussion by ReiTheGoat01** [Stardust-Labs-MC/StardustMC#46] [Ideas]
These could be anything you wish to see or participate in and the rewards you would want from them. (For example a spleef tournament or bingo event in a RW with prizes being physical like netherite or cosmetic like a title that is worn by the most recent winner)

**Comment by ReiTheGoat01** [comment]
I would really like to see a Bingo event using RW as a playing field. Although maybe for this type of event inventories shouldn't be carried over so no one player has an advantage, and all players can play equally.

Additionally other events like a building competition or hide and seek in the shopping district could be fun.

**Comment by CodingPanda0623** [comment]
A spleef tournament sounds nice if someone sets up haste 2 beacons and basalt generators. Personally, I’d like to experience something like the end fight again, where everyone needed to work together to kill the ender dragon, but maybe this time put a couple buffed withers for us to fight (I would suggest a warden but given its single target tendencies I don’t think a fight with them are gonna last long if some guys bait them and run). I also had an idea for a sumo arena so maybe we could have some sort of event there. Currently, the only “event” we could host is a pvp event sorta.

**Comment by EshXKun** [comment]
Please consider starz as a reward too🤣🤣

**Reply by CodingPanda0623** [reply]
if I was richer I would sponsor an event lmao but unfortunately I'm poor until I sell gold and stuff

**Comment by catter1** [comment]
Feel free to suggest things that aren't technically possible in StardustMC, but would still be tons of fun. We can always create a new "server" just for events (which has been planned for the start) and reset it each time with a new event.
